source/form.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Academy: Мишка</title>
+</head>
+
+<body>
+  <header class="main-header">
+    <a class="main-header__logo" href="index.html">
+      <img class="main-header__logo-image" src="img/logo.svg" width="86" height="35" alt="DIY магазин «Мишка».">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"></button>
+      <span class="visually-hidden">Открыть меню.</span>
+      <div class="main-nav-wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a class="site-list__link" href="catalog.html">
+              Каталог товаров
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">
+              Вязание на заказ
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link site-list__link--search" href="#">
+              Поиск по сайту
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link site-list__link--cart" href="#">
+              Корзина: пока пуста
+            </a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+  <main class="main-wrapper">
+    <h1 class="hero-title">Вязание на заказ</h1>
+    <section class="forms">
+      <p class="forms__customization-text">Мы будем рады воплотить в жизнь ваши пожелания! Заполните простую форму
+        заказа и мы свяжемся с вами, чтобы уточнить детали.</p>
+      <h2 class="visually-hidden">Настройка заказа.</h2>
+      <form class="form" id="form-area" method="get" action="https://echo.htmlacademy.ru">
+        <fieldset class="form__type form-section">
+          <legend class="form__label--primary">Тип</legend>
+          <ul class="form__form-list">
+            <li class="form-list-item">
+              <input class="type__radio" type="radio" name="Тип" value="Аксессуар для интерьера" id="accesory" checked>
+              <label class="type__label form-text" for="accesory">Аксессуар для интерьера</label>
+            </li>
+            <li class="form-list-item">
+              <input class="type__radio" type="radio" name="Тип" value="Детская игрушка" id="toy">
```

Comment:
name только латинницей и без спецсимволов.

```suggestion
              <input class="type__radio" type="radio" name="type" value="Детская игрушка" id="toy">
```